unit_tests/code_gen_tests.cc

```diff
@@ -0,0 +1,300 @@
+// Copyright (c) 2022-present, IO Visor Project
+// SPDX-License-Identifier: Apache-2.0
+
+#include <filesystem>
+#include <fstream>
+
+#include <gtest/gtest.h>
+
+#include "bpf_assembler.h"
+#include "test_helpers.h"
+extern "C"
+{
+#include "ubpf.h"
+}
+
+std::map<std::string, std::string> unsupported_tests {
+#if defined(__aarch64__)
+    {"mod-by-zero-reg.data", "JIT"},
+    {"div-by-zero-reg.data", "JIT"},
+    {"div-by-zero-imm.data", "JIT"},
+    {"mod-by-zero-reg.data", "JIT"},
+    {"mod-by-zero-reg.data", "JIT"},
+    { "mod-by-zero-reg.data", "JIT"}, 
+    {"div-by-zero-reg.data", "JIT"}, 
+    {"div-by-zero-imm.data", "JIT"}, 
+    {"div64-by-zero-imm.data", "JIT"}, 
+    {"mod64-by-zero-imm.data", "JIT"}, 
+    {"mod-by-zero-imm.data", "JIT"}, 
+    {"div64-by-zero-reg.data", "JIT"}, 
+    {"mod64-by-zero-reg.data", "JIT"}, 

```

Comment:
These should be removable if #123 is merged.

Reply:
Thanks. Done.